app/graphql/mutations/remove_asset_from_consignment_submission.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Mutations
+    class RemoveAssetFromConsignmentSubmission < Mutations::BaseMutation
+        argument :gemini_token, String, required: true
+        argument :submissionID, ID, required: true
+        argument :sessionID, String, required: false
+        argument :asset_type, String, required: false
+        argument :filename, String, required: false
+        argument :size, String, required: false
```

Comment:
Looks like there are a few unnecessary parameters: asset_type, filename, size. I also think we can use assetID instead of gemini_token and submissionID, in this case, we will load the exact asset then load their submission to validate the user, and if all is good we will be able to remove the asset.

Reply:
@Gon4ar5 Could you please help with that?